src/components/PickerTemplate/PickerTemplate.css

```diff
@@ -12,5 +12,5 @@
   font-family: 'rennerbold';
   color: #717171;
   text-align: center;
-  margin-bottom: 30px;
+  margin-bottom: 20px;
```

Comment:
you realize this will affect all "Picker" components? Is that the intended effect?

Reply:
yes. 30px is too much